pkgs/misc/pywal/default.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs ? import <nixpkgs> {} }:
+let
+  py = pkgs.python36Packages;
+in py.buildPythonPackage rec {
+  name = "pywal";
+  version = "2.0.4";
+  src = builtins.fetchTarball {
+    url = "https://github.com/dylanaraps/pywal/archive/${version}.tar.gz";
+  };
```

Comment:
Use `fetchFromGitHub` here and add a checksum.

Reply:
You could also use `fetchPypi`: https://pypi.org/project/pywal/.